pkg/subctl/operator/install/crds/ensure.go

```diff
@@ -37,7 +37,7 @@ func updateOrCreateCRD(clientSet *clientset.Clientset, crd *apiextensionsv1beta1
 
 	_, err := clientSet.ApiextensionsV1beta1().CustomResourceDefinitions().Create(crd)
 	if err == nil {
-		return true, err
+		return true, nil
```

Comment:
This is indicative that the existing unit tests were not enough, I will enhance coverage as part of this PR